### DISCUSSION TITLE
Integration with org mode?

### DISCUSSION THREAD

**Discussion by librarianmage** [tectonic-typesetting/tectonic#928] [Q&A]
Has anyone integrated tectonic to be used as the export for org mode? I'm trying to do that but am struggling.

**Comment by pkgw** [comment]
I'm curious about what exactly you're trying to achieve. Does org mode "support" TeX in some sense?

**Reply by librarianmage** [reply]
Indeed! Emacs can [export](https://orgmode.org/manual/LaTeX-Export.html) documents into LaTeX / pdfs

**Comment by pkgw** [comment]
Well, based on a cursory scan of what looks like the org-mode latex source code, I don't see any huge red flags. It looks like the `-p` option to print the usual TeX engine output or `-k` to keep intermediate files might either, or both, be needed. Is there a specific error message or problem that you're hitting?

**Comment by librarianmage** [comment]
it seems like my issues stem from my current configuration being rooted in
LuaLaTeX; I will report back if that changes things.

**Comment by zzamboni** [comment]
FWIW, I am using Tectonic to render a LaTeX document generated by Org-mode, and it works well. In case you are interested in an example: https://github.com/zzamboni/vita

**Comment by dragove** [comment]
try change org-latex-pdf-process varaiable in emacs

```elisp
(setq
     org-latex-pdf-process
     '("tectonic -X compile --outdir=%o -Z shell-escape -Z continue-on-errors %f")))
```
